docs/README.md

```diff
@@ -9,12 +9,21 @@ Packages are available at: https://github.com/orgs/terrafx/packages or via the N
 
 ## Table of Contents
 
+* [Benchmarks](#benchmarks)
 * [Code of Conduct](#code-of-conduct)
 * [License](#license)
 * [Contributing](#contributing)
 * [Goals](#goals)
 * [Languages and Frameworks](#languages-and-frameworks)
 
+### Benchmarks
```

Comment:
Benchmarks needs to be below Code of Conduct/License. I think probably last in the list.

Reply:
Hmm. May I ask why? Isn't the meaning of a repo more important than legal issues?

Reply:
My own experience is that rules for interacting with the repo should be first. If you don't agree with the rules, you shouldn't contribute

It minorly helps dealing with trolls and stuff and the table of contents gives you a quick overview/access to the other sections anyways.